modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,98 @@
+var username = $("#navbar_profile_link").text().substr(1);
+if (document.getElementById("navigation_top")){
+    var Credits = document.getElementById("navigation_top").outerHTML;
+}
+if (document.getElementById("coins_top")){
+    var Coins = document.getElementById("coins_top").outerHTML;
+}
+var credits = document.getElementById("navigation_top");
+var coins = document.getElementById("coins_top");
+var gesamtcredits = "undefined";
+var dienstgrad = "undefined";
+var next_level = "undefined";
+
+function lightbox_open() {
+    lightboxOpen();
+}
+
+function setCookies() {
```

Comment:
Grundsätzlich finde ich die Herangehensweise mit dem Cookie nicht optimal gelöst. In dieser Funktion ist jedoch noch mehr Verbesserungsbedarf (z.B. funktioniert das nur mit dem deutschen Spiel.)